scripts/nn/test/grad_check.dml

```diff
@@ -616,6 +617,114 @@ conv2d_simple = function() {
   }
 }
 
+conv2d_transpose = function() {
+  /*
+   * Gradient check for the 2D convolution transpose layer.
+   */
+  print("Grad checking the 2D convolution transpose layer with L2 loss.")
+
+  N = 1
+  C = 1
+  Hin = 3
+  Win = 3
+  F = 1
+  Hf = 3
+  Wf = 3
+  stride = 2
+  pad = 1
+  out_pad = 1
+
+  X = rand(rows=N, cols=C*Hin*Win)
+
+  [W,b] = conv2d_transpose::init(F, C, Hf, Wf)
+  
+  [out, Hout, Wout] = 
+ 	conv2d_transpose::forward(X, W, b, C, Hin, Win, Hf, Wf, stride, stride, pad, pad, out_pad, out_pad)
```

Comment:
Can you update the formatting for this function so that no lines, such as this one, are longer than 100 characters?  In addition to keep docs <=72, I've kept all lines <=100, based on other style guides.